src/quests/fetchQuestData.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios'
+
+/**
+ * Fetches quest data from RabbitHole API and extracts action parameters.
+ * @param {string} uuid The UUID of the quest.
+ * @returns {Promise<any>} The action parameters extracted from the quest data.
+ */
+export async function fetchQuestActionParams(uuid: string): Promise<any> {
+  const endpoint = `https://api.rabbithole.gg/v1.2/quest/public/${uuid}`
```

Comment:
Would add a TODO or open up an issue to update this endpoint to a Boost API in the near future